frontend/src/types/AuthTypes.ts

```diff
@@ -11,3 +11,8 @@ export type DecodedJWT =
   | string
   | null
   | { [key: string]: unknown; exp: number };
+
+export enum Role {
```

Comment:
why are we changing this from a type to an `enum`?

Reply:
to use `Role` as a value (i.e. check if role == Role.ADMIN) - but i will remove this since the check is not necessary